src/lib/ndpi_main.c

```diff
@@ -2308,6 +2304,10 @@ static void ndpi_init_protocol_defaults(struct ndpi_detection_module_struct *ndp
 			  "BFCP", NDPI_PROTOCOL_CATEGORY_VIDEO,
 			  ndpi_build_default_ports(ports_a, 0, 0, 0, 0, 0) /* TCP */,
 			  ndpi_build_default_ports(ports_b, 0, 0, 0, 0, 0) /* UDP */);
+  ndpi_set_proto_defaults(ndpi_str, 0 /* encrypted */, 0 /* nw proto */, NDPI_PROTOCOL_FUN, NDPI_PROTOCOL_IQIYI,
+			  "iQIYI", NDPI_PROTOCOL_CATEGORY_VIDEO,
```

Comment:
`NDPI_PROTOCOL_CATEGORY_STREAMING`? Not a big difference, but I'd prefer to have the same category here and in the domains list

Reply:
> `NDPI_PROTOCOL_CATEGORY_STREAMING`? Not a big difference, but I'd prefer to have the same category here and in the domains list

Apparently I missed it, I'll fix it now.